src/routes/docs/advanced/self-hosting/platforms/google-cloud/+page.markdoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: article
+title: Google Cloud deployment
+description: Deploy Appwrite on Google Cloud Platform using Cloud Run or Compute Engine. Learn how to set up a production-ready Appwrite instance with automatic scaling.
+---
+
+Google Cloud Platform (GCP) offers multiple deployment options for Appwrite. This guide covers the most effective deployment methods for different use cases.
+
+# Deployment options {% #deployment-options %}
+
+Choose the deployment method that best fits your requirements:
+
+{% table %}
+* Service
+* Best For
+* Complexity
+* Scaling
+---
+* **Cloud Run**
+* Serverless, automatic scaling
+* Low
+* Automatic
+---
+* **Compute Engine**
+* Custom configurations, full control
+* Medium
+* Manual
+---
+* **App Engine**
+* Managed platform, simple deployment
+* Low
+* Automatic
+{% /table %}
+
+# Cloud Run deployment {% #cloud-run %}
+
+Deploy Appwrite using Google Cloud Run for a serverless, fully managed experience with automatic scaling.
+
+{% info title="Compatibility note" %}
+Appwrite uses MariaDB by default. Running against Cloud SQL for MySQL 8.x is unsupported and may break schema or SQL dialect assumptions. Test thoroughly in a staging environment or deploy MariaDB yourself on Compute Engine instead.
+{% /info %}
+
+For detailed Cloud Run deployment instructions, follow the [general installation guide](/docs/advanced/self-hosting/installation) and adapt the Docker commands for your Cloud Run environment.
```

Comment:
I'm not sure about mentioning all of these that may not work.

Google Cloud Run is similar to a AWS Lambda. We should be focusing on Compute Engine as that's similar to AWS' EC2 and DO's droplets.

Reply:
Got it. Updated